lib/utils/nightwatchInterfaces.ts

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
As these types are commonly used throughout the codebase, it would make sense to move them under a `types` folder. My proposed folder structure would be as follows:
```
- utils
  - types
    - index.ts
```

Thought?

Reply:
Looks good to me.